scip.proto

```diff
@@ -384,79 +384,93 @@ enum DiagnosticTag {
 // Feel free to send a pull-request to add missing programming languages.
 enum Language {
   UnspecifiedLanguage = 0;
-  Java = 1;
-  Scala = 2;
-  Kotlin = 3;
-  Go = 4;
-  TypeScript = 5;
-  JavaScript = 6;
-  Python = 7;
-  Ruby = 8;
-  C_Sharp = 9;
-  C_PlusPlus = 10;
-  Objective_C = 11;
-  Swift = 12;
-  PHP = 13;
-  CSS = 14;
-  C = 15;
-  R = 16;
-  Shell = 17;
-  PowerShell = 18;
-  Rust = 19;
-  Dart = 20;
-  Matlab = 21;
-  VBA = 22;
-  Ada = 23;
-  Idris = 24;
-  Visual_Basic = 25;
-  Abap = 26;
-  Lua = 27;
-  Groovy = 28;
-  Perl = 29;
-  Julia = 30;
-  Cobol = 31;
-  Haskell = 32;
-  Delphi = 33;
-  Assembly = 34;
-  SQL = 35;
-  Prolog = 36;
-  SAS = 37;
-  Scratch = 38;
-  Fortran = 39;
-  Lisp = 40;
-  PL_SQL = 41;
-  YAML = 42;
-  JSON = 43;
-  TOML = 44;
-  Jsonnet = 45;
-  Awk = 46;
-  Hack = 47;
-  Racket = 48;
-  Flow = 49;
-  Vue = 50;
-  Elixir = 51;
-  WindowsBat = 52;
-  BibTeX = 53;
-  Clojure = 54;
-  Coffeescript = 55;
-  Diff = 56;
-  Dockerfile = 57;
-  Erlang = 58;
-  F_Sharp = 59;
-  Git = 60;
-  Handlebars = 62;
-  HTML = 63;
-  Ini = 64;
-  LaTeX = 65;
-  Less = 66;
-  Makefile = 67;
-  Markdown = 68;
-  Objective_C_PlusPlus = 69;
-  Perl6 = 70;
-  Razor = 71;
-  SCSS = 72;
-  Sass = 73;
-  ShaderLab = 74;
-  TeX = 75;
-  XSL = 76;
+  CSharp = 1;
+  Swift = 2;
+  Dart = 3;
+  Kotlin = 4;
+  Scala = 5;
+  Java = 6;
+  Groovy = 7;
+  Clojure = 8;
+  CommonLisp = 9;
+  Scheme = 10;
+  Racket = 11;
+  Lua = 12;
+  Perl = 13;
+  Raku = 14;
+  Python = 15;
+  Ruby = 16;
+  Elixir = 17;
+  Erlang = 18;
+  PHP = 19;
+  Hack = 20;
+  Coffeescript = 21;
+  JavaScript = 22;
+  TypeScript = 23;
+  Flow = 24;
+  Vue = 25;
+  CSS = 26;
+  Less = 27;
+  Sass = 28;
+  SCSS = 29;
+  HTML = 30;
+  XML = 31;
+  XSL = 32;
+  Go = 33;
+  C = 34;
+  CPlusPlus = 35;
+  ObjectiveC = 36;
+  ObjectiveCPlusPlus = 37;
+  Zig = 38;
+  Ada = 39;
+  Rust = 40;
+  OCaml = 41;
+  FSharp = 42;
+  SML = 43;
+  Haskell = 44;
+  Agda = 45;
+  Idris = 46;
+  Coq = 47;
+  Lean = 48;
+  APL = 49;
+  Dyalog = 50;
+  J = 51;
+  Matlab = 52;
+  Wolfram = 53;
+  R = 54;
+  Julia =  55;
+  Fortran = 56;
+  Delphi = 57;
+  Assembly = 58;
+  COBOL = 59;
+  ABAP = 60;
+  SAS = 61;
+  Razor = 62;
+  VisualBasic = 63;
+  Shell = 64;
+  Fish = 65;
+  Awk = 66;
+  PowerShell = 67;
+  WindowsBat = 68;
+  SQL = 69;
+  PLSQL = 70;
+  Prolog = 71;
+  Ini = 72;
+  TOML = 73;
+  YAML = 74;
+  JSON = 75;
+  Jsonnet = 76;
+  Nix = 77;
+  Bazel = 78;
```

Comment:
Should this be Skylark? BUILD files and `.bzl` files are written in Skylark. There's no "Bazel" language to the best of my understanding